javascript-modal/main.js

```diff
@@ -0,0 +1,22 @@
+var count = false;
+var $blue = document.querySelector('.blue');
+var $modal = document.querySelector('.modal');
+var $red = document.querySelector('.red');
+var $background = document.querySelector('.background');
+$blue.addEventListener('click', popup);
+function popup() {
+  count = true;
+  if (count === true) {
+    $modal.className = 'modal on';
+    $background.className = 'background dark';
+  }
+}
+
+$red.addEventListener('click', popdown);
+function popdown() {
+  count = false;
+  if (count === false) {
```

Comment:
This conditional isn't necessary. You are assigning a `false` value on the line before the conditional, so this conditional would always execute.

Reply:
yeah that was dumb.